### ISSUE TITLE
Question about depth regression

### ISSUE THREAD

**Issue by pnpmpnp** [mileyan/AnyNet#44]
I was looking at the code of AnyNet and had a question.
when you do disparity_regression in AnyNet, you do it for "-cost", is there a reason for doing it this way ?
> "pred_low_res = disparityregression2(-self.maxdisplist[scale]+1, self.maxdisplist[scale], stride=1)(F.softmax(-cost, dim=1))"

**Comment by POLUT-W** [comment]
It may be to try to obtain the minimum cost value. By the way, have you ever reproduced this network? Why does my loss keep hovering around 10
